config/environments/development.rb

```diff
@@ -3,6 +3,14 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Whitelist lab hostname
+  config.hosts << "rails-canary-lab.aws.cru.org"
+
+  # Web Console
+  # https://github.com/rails/web-console
+  config.web_console.whiny_requests = false
+  config.web_console.permissions = "10.0.0.0/8"
+
```

Comment:
Are these purely related to the lab instance?

Reply:
The config.hosts was due to the dns remapping that's enabled in development mode, which is what `lab` ran as.

I believe I set the others based on console/log output warning about them not being set in the lab/dev testing. I don't think it hurts anything to leave them.